Makefrag

```diff
@@ -35,8 +35,8 @@ $(FIRRTL_JAR): $(shell find $(base_dir)/firrtl/src/main/scala -iname "*.scala")
 
 src_path := src/main/scala
 resources := $(base_dir)/src/main/resources
-csrc := $(resources)/csrc/
-vsrc := $(resources)/vsrc/
+csrc := $(resources)/csrc
```

Comment:
My personal preference is to leave trailing slashes so even slashes are forgotten later, you're still safe. It's not wrong to remove them, though. Is there a good reason to remove them?

Reply:
I check most variables in Makefiles of the project, which have no tailing slashes. So I think it is better all variable follow the same rule.